llvm/cmake/config-ix.cmake

```diff
@@ -257,6 +257,26 @@ else()
   set(LLVM_ENABLE_TERMINFO 0)
 endif()
 
+#Check for icu.
+if(LLVM_ENABLE_ICU)
+  find_package(ICU COMPONENTS uc i18n)
```

Comment:
https://discourse.llvm.org/t/rfc-enabling-fexec-charset-support-to-llvm-and-clang-reposting/71512/30 distinguishes between statically and dynamically linking against ICU; which one does this do?

Reply:
I think I will need more work for this, thanks for catching! In my local testing I only had the shared library but I think the current implementation will allow static linking.

Reply:
I looked into implementing this by setting the following cmake variable CMAKE_FIND_LIBRARY_SUFFIXES so that it will only find libraries ending in .so for ICU. Please let me know if there is a better way to do this. Thanks!

Reply:
I can't think of anything better.

This won't work on Windows.